src/StackManagerVersion.h.in

```diff
@@ -12,3 +12,5 @@
 //
 
 const QString BUILD_VERSION = "@BUILD_SEQ@";
+const QString URL_BASE = "@BASE_URL@";
+const bool PR_BUILD = @PR_BUILD@;
```

Comment:
This is really not a .h until cmakes takes the .h.in and turns it into a .h, filling in the variables enclosed in @ @